tests/confs/mockduo_space_users.conf

```diff
@@ -0,0 +1,6 @@
+[duo]
+ikey = DIXYZV6YM8IFYVWBINCA
+skey = yWHSMhWucAcp7qvuH3HWTaSaKABs8Gaddiv1NIRo
+host = localhost:4443
+cafile = certs/mockduo-ca.pem
+groups = users\ with\ spaces
```

Comment:
I added in tests for the other groups you mentioned and changed "no_space_here" to "no_spaces\here" so it also tests backslashes in the group name. I also removed the second comma so it will test space delimeter group names at the same time.